suite-native/intl/src/localeSlice.ts

```diff
@@ -28,7 +28,7 @@ export const localeSlice = createSlice({
     },
 });
 
-export const selectLanguage = (state: LocaleSliceRootState) => state.locale.language;
+export const selectUserSelectedLocale = (state: LocaleSliceRootState) => state.locale.language;
```

Comment:
Since all of the hooks and selectors changed name, I believe `state.locale.language` should be `state.locale.locale`